scripts/benchmark/query/load/tpch.sh

```diff
@@ -0,0 +1,125 @@
+    c_mktsegment  STRING not null,
+    c_comment     STRING not null
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS lineitem (
+    l_orderkey    BIGINT not null,
+    l_partkey     BIGINT not null,
+    l_suppkey     BIGINT not null,
+    l_linenumber  BIGINT not null,
+    l_quantity    DOUBLE not null,
+    l_extendedprice  DOUBLE not null,
+    l_discount    DOUBLE not null,
+    l_tax         DOUBLE not null,
+    l_returnflag  STRING not null,
+    l_linestatus  STRING not null,
+    l_shipdate    DATE not null,
+    l_commitdate  DATE not null,
+    l_receiptdate DATE not null,
+    l_shipinstruct STRING not null,
+    l_shipmode     STRING not null,
+    l_comment      STRING not null
+  );
+SQL
+
+# create tpch tables
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS nation (
+    n_nationkey  INTEGER not null,
+    n_name       STRING not null,
+    n_regionkey  INTEGER not null,
+    n_comment    STRING
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS orders (
+    o_orderkey       BIGINT not null,
+    o_custkey        BIGINT not null,
+    o_orderstatus    STRING not null,
+    o_totalprice     DOUBLE not null,
+    o_orderdate      DATE not null,
+    o_orderpriority  STRING not null,
+    o_clerk          STRING not null,
+    o_shippriority   INTEGER not null,
+    o_comment        STRING not null
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS partsupp (
+    ps_partkey     BIGINT not null,
+    ps_suppkey     BIGINT not null,
+    ps_availqty    BIGINT not null,
+    ps_supplycost  DOUBLE not null,
+    ps_comment     STRING not null
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS part (
+    p_partkey     BIGINT not null,
+    p_name        STRING not null,
+    p_mfgr        STRING not null,
+    p_brand       STRING not null,
+    p_type        STRING not null,
+    p_size        INTEGER not null,
+    p_container   STRING not null,
+    p_retailprice DOUBLE not null,
+    p_comment     STRING not null
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS region (
+    r_regionkey  INTEGER not null,
+    r_name       STRING not null,
+    r_comment    STRING
+  );
+SQL
+
+cat <<SQL | bendsql query
+  CREATE TABLE IF NOT EXISTS supplier (
+    s_suppkey     BIGINT not null,
+    s_name        STRING not null,
+    s_address     STRING not null,
+    s_nationkey   INTEGER not null,
+    s_phone       STRING not null,
+    s_acctbal     DOUBLE not null,
+    s_comment     STRING not null
+  );
+SQL
+
+for t in customer lineitem nation orders partsupp part region supplier; do
+  echo "loading into $t ..."
+  cat <<SQL | bendsql query
+COPY INTO $t FROM 's3://repo.databend.rs/tpch10/${t}/'
+credentials=(aws_key_id='$AWS_KEY_ID' aws_secret_key='$AWS_SECRET_KEY') pattern ='${t}.*'
+file_format=(type='CSV' field_delimiter='|' record_delimiter='\\n' skip_header=0);
+SQL
+  cat <<SQL | bendsql query
```

Comment:
Should do an `analyze table ` for tables: `customer lineitem nation orders partsupp part region supplier` after load data? @xudong963

Reply:
Yes